src/widgets/settingspages/SettingsPage.cpp

```diff
@@ -21,36 +20,36 @@ bool filterItemsRec(QObject *object, const QString &query)
             widget->update();
         };
 
-        if (auto x = dynamic_cast<SCheckBox *>(child); x)
+        if (auto xx = dynamic_cast<SCheckBox *>(child); xx)
```

Comment:
weird variable names again

Reply:
Do you have a preference on name?

Reply:
(FWIW, I tend to use variable names that are at least 2 letters so that they're easier to search for)

Reply:
`checkBox`, `label`, `comboBox`?